### PR TITLE
Add cookiecutter

### PR DESCRIPTION
**Pull request by kragniz**

### CODE REVIEW

**Comment by kragniz** [comment]
@domenkozar any idea why this is trying to install flask? I'm probably doing something dumb, but I don't see what

**Comment by domenkozar** [comment]
@kragniz I think it wants it for testing.

**Comment by kragniz** [comment]
I'm not seeing any references to flask in:

    $ grep -ri flask ~/git/cookiecutter

Is there somewhere else it could be picking it up?

**Comment by domenkozar** [comment]
Sorry I'm a bit busy, give me a few days :)

File: pkgs/development/tools/misc/cookiecutter/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  version = "1.3.0";
+  name = "cookiecutter-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/audreyr/cookiecutter/archive/${version}.tar.gz";
+    md5 = "378670fe456957eb3c27ddaef60b2b24";
+  };
```
Comment (FRidh) [inline pkgs/development/tools/misc/cookiecutter/default.nix]: could you use sha256 here? 

Also, in my opinion, this should be moved to `python-packages.nix`.
You can still put an entry in `all-packages.nix`, e.g. `cookiecutter = pythonPackages.cookiecutter;`

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -1490,6 +1490,27 @@ in modules // {
   };
 
 
+  binaryornot = buildPythonPackage rec {
+    name = "binaryornot-${version}";
+    version = "0.4.0";
+
+    doCheck = false;
+
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: why do you disable tests? Please add the reason here or to the commit msg

**Comment by FRidh** [comment]
Could you also rename the commits according to how it's described in the [guide](http://nixos.org/nixpkgs/manual/#chap-submitting-changes), e.g. `pythonPackages.binaryornot: init at 0.4.0`

**Comment by FRidh** [comment]
@kragniz What is the status of this PR? Does it still try to install Flask? Did you test whether everything builds with `nox-review wip` or `nox review pr 11589`?

**Comment by kragniz** [comment]
After #12405 was closed, this looks like it's working okay.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -3130,6 +3151,30 @@ in modules // {
...
+    };
+
+    buildInputs = with self; [ itsdangerous  ];
+    propagatedBuildInputs = with self; [
+          jinja2 future binaryornot click whichcraft ruamel_yaml ];
+
+    meta = {
+      homepage = https://github.com/audreyr/cookiecutter;
+      description = "A command-line utility that creates projects from project templates";
+      license = "BSD";
```
Comment (globin) [inline pkgs/top-level/python-packages.nix]: Please use the attribute from `stdenv.lib.licenses`

Comment (kragniz) [inline pkgs/top-level/python-packages.nix]: Done.

**Comment by globin** [comment]
Thanks :+1:

**Pull request merged**